.github/workflows/ci.yml

```diff
@@ -1,38 +1,26 @@
 name: CI
 
 on:
+  workflow_dispatch:
   push:
     branches:
       - master
+      - run-ci-on-windows
```

Comment:
Is the PR trigger insufficient?  Is this expected to remain and get merged?

Reply:
No, it's a leftover. I needed it to trigger the CI on my own repo, for this branch. It needs to go now.